extensions/guacamole-auth-radius/src/main/java/org/apache/guacamole/auth/radius/conf/RadiusAuthenticationProtocol.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.apache.guacamole.auth.radius.conf;
+
+/**
+ * This enum represents supported RADIUS authentication protocols for
+ * the guacamole-auth-radius extension.
+ */
+public enum RadiusAuthenticationProtocol {
+    
+    // Password authentication protocol
+    PAP("pap"),
+    
+    // Challenge-Handshake AUthentication Protocol
+    CHAP("chap"),
+    
+    // Microsoft CHAP version 1
+    MSCHAPv1("mschapv1"),
+    
+    // Microsoft CHAP version 2
+    MSCHAPv2("mschapv2"),
+    
+    // Extensible authentication protocol with MD5 hashing.
+    EAP_MD5("eap-md5"),
+
+    // Extensible authentication protocol with TLS
+    EAP_TLS("eap-tls"),
+
+    // Extensible authentication protocol with Tunneled TLS
+    EAP_TTLS("eap-ttls");
+
+    // Store the string value used in the configuration file.
+    private final String strValue;
+    
+    /**
+     * Create a new RadiusAuthenticationProtocol object having the
+     * given string value.
+     * 
+     * @param strValue
+     *     The string value of the protocol.
+     */
+    public RadiusAuthenticationProtocol(String strValue) {
+        this.strValue = strValue;
+    }
+    
+    @Override
+    public String toString() {
```

Comment:
Though it's not normally needed to document `toString()`, I would suggest doing so here since we are depending on this function having specific behavior (returning the value used by both `guacamole.properties` and specific parts of JRadius).

Something like:

```java
/**
 * {@inheritDoc}
 * <p>
 * Doooocuuumentaaationnn...
 */
```

would probably do well.

The alternative would be to not use `toString()` but our own function with well defined semantics regarding the return value and `guacamole.properties` and JRadius.

Reply:
Doc'd.